pegasus/sites.v3/hourofcode.com/public/css/action-blocks.scss

```diff
@@ -59,6 +63,7 @@
   flex-wrap: wrap;
   flex-direction: column;
   padding: 1.25rem;
+  text-align: initial;
```

Comment:
This add RTL language support.

Reply:
TIL about `initial` as a `text-align` option!